metrics_utility/automation_controller_billing/dataframe_engine/dataframe_collection_status.py

```diff
@@ -12,7 +12,7 @@ def build_dataframe(self):
         for date in self.dates():
             for data in self.extractor.iter_batches(date=date):
                 batch = data['data_collection_status']
-                if batch.empty:
+                if batch is None or batch.empty:
```

Comment:
(wondering if we need these, since the extractor now returns an empty dataframe - goes for all the None checks and `.get` calls)